transaction/core/src/token.rs

```diff
@@ -18,7 +18,7 @@ impl From<u32> for TokenId {
 
 impl fmt::Display for TokenId {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        write!(f, "{:?}", self)
+        write!(f, "{}", self.0)
```

Comment:
In another PR it was decided that the `Display` implementation should not wrap the value with any context, we have `Debug` for that.